runtime/debugger.c

```diff
@@ -363,14 +340,11 @@ void caml_debugger_code_unloaded(int index)
 
   cf = (struct code_fragment *) caml_code_fragments_table.contents[index];
 
-  for (i = 0; i < breakpoints_table.size; i++) {
-    bpti = (struct breakpoint *) breakpoints_table.contents[i];
-    if ((char*) bpti->pc >= cf->code_start && (char*) bpti->pc < cf->code_end) {
-      caml_ext_table_remove(&breakpoints_table, bpti);
-      /* caml_ext_table_remove has shifted the next element in place
-         of the one we just removed. Decrement i for the next
-         iteration. */
-      i--;
+  for (e = event_points_table.forward[0]; e != NULL; e = f) {
+    f = e->forward[0];
+    pc = (char *) e->key;
+    if (pc >= cf->code_start && pc < cf->code_end) {
+      caml_skiplist_remove(&event_points_table, (uintnat) pc);
     }
   }
 }
```

Comment:
Note that this function `caml_debugger_code_unloaded` is unused: there is no other occurrence in the whole source tree.  Since it is somewhat tricky, can I have permission to remove it?  Asking @jhjourdan.